chapters/0-introduction.tex

```diff
@@ -342,16 +302,88 @@ \section{Documenting data work with standardized code}
+Resources developed and tested in DIME are converted into public goods
+for the global research community, through open-access trainings and open-source tools.
+The DIME Analytics Resource Directory appendix provides an introduction to public materials.
+
+DIME Analytics has invested many hours over the past years
+learning from data work across DIME's portfolio,
+identifying inefficiencies and barriers to success,
+developing tools and trainings, and standardizing best-practice workflows adopted in DIME projects.
+It has also invested significant energy in the language and materials
+used to teach these workflows to new team members,
+and, in many cases, in software tools that support these workflows explicitly.
+DIME team members often work on diverse portfolios of projects
+with a wide range of teammates, and we have found
+that standardizing core processes across all projects
+results in higher-quality work with fewer opportunities to make mistakes.
+In that way, the Analytics team is DIME's method of institutionalizing
+tools and practices, developed and refined over time,
+that give the department a common base of knowledge and practice.
+In 2018, for example, DIME adopted universal reproducibility checks
+conducted by the Analytics team;
+the lessons from this practice helped move the DIME team
+from where 50\% of submitted papers in 2018
+required significant revision to pass
+to where 64\% of papers passed in 2019 without revision required.
+
+
+\section{Looking ahead}
 While adopting the workflows and mindsets described in this book requires an up-front cost,
 it will save you (and your collaborators) a lot of time and hassle very quickly.
 In part this is because you will learn how to implement essential practices directly;
 in part because you will find tools for the more advanced practices;
 and most importantly because you will acquire the mindset of doing research with a high-quality data focus.
-We hope you will find this book helpful for accomplishing all of the above,
-and that mastery of data helps you make an impact.
+
+For some readers, the amount of new tools and practices recommended in this book may seem daunting.
+We know from experience at DIME that full-scale adoption is possible;
+in the last few years, the full DIME portfolio has transitioned to 
+transparent and reproducible workflows, with a fair share of hiccups along the way.
+The authors of this book supported that at-scale transition,
+and we hope that by sharing our lessons learned and resources, 
+the learning curve for readers will be less steep. 
+In the summary boxes at the beginning of each chapter, 
+we provide a list of the key tools and resources to help readers prioritize.
+We will also offer 'second-best' practices in many cases, 
+suggesting easy-to-implement suggestions to increase transparency and reproducibility,
+in cases where full-scale adoption of the recommended workflows is not feasible.   
```

Comment:
```suggestion
in cases where full-scale adoption of the recommended workflows is not immediately feasible.
In fact, we encourage teams to adopt one new practice at a time 
rather than rebuild their whole workflow from scratch right away.
```